lca-ai-stack/deployment/lca-ai-stack.yaml

```diff
@@ -19,6 +19,10 @@ Transform: AWS::Serverless-2016-10-31
 Description: Amazon Transcribe Live Call Analytics with Agent Assist - LCA AI Stack
 
 Parameters:
+  ExpirationInDays:
+    Type: Number
+    Description: "Number of days after which bucket objects will be deleted."
```

Comment:
Add clarification: Number of days after which bucket objects will be deleted *from the Recordings bucket*